src/discussions/discussions-home/InformationsBanner.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Hyperlink, PageBanner } from '@edx/paragon';
+
+import { selectUserIsStaff, selectUserRoles } from '../data/selectors';
+import messages from '../messages';
+
+function InformationBanner({
+  intl,
+}) {
+  const [showBanner, setShowBanner] = useState(true);
+  const userRoles = useSelector(selectUserRoles);
+  const isAdmin = useSelector(selectUserIsStaff);
+  const learnMoreLink = 'https://openedx.atlassian.net/wiki/spaces/COMM/pages/3509551260/Overview+New+discussions+experience';
+  const learnerFeedbackLink = 'https://forms.gle/foGYYQjHZfWTzBiR8';
+  const staffFeedbackLink = 'https://forms.gle/APn3k39QHmieLfJaA';
```

Comment:
suggestion: we should move these feedback forms into the env variables. This repo this public so maybe someone can use this form for fun. which will affect feedback data authenticity.

Reply:
I agree